### DISCUSSION TITLE
Question about remote image format

### DISCUSSION THREAD

**Discussion by shellyfengfeng** [OpenAMP/open-amp#498] [Q&A]
The remoteproc component currently supports Executable and Linkable Format (ELF) for the remote firmware, however, could it support other image formats like binary format? 
Thanks.

**Comment by tnmysh** [comment]
Hi,

Only ELF is supported format.
any other format for remote processor firmware is not supported yet.

Thanks.

**Comment by arnopo** [comment]
@shellyfengfeng 
you should be able to implement your binary loader using https://github.com/OpenAMP/open-amp/blob/main/lib/include/openamp/remoteproc_loader.h API
